### PR TITLE
UIF-305 Implement Keyboard shortcuts modal

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
https://issues.folio.org/browse/UIF-305

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -5,63 +5,108 @@ import {
...
         </HasCommand>
       </CommandList>
-    );
-  }
-}
+      {isOpen && (
+        <AcqKeyboardShortcutsModal
+          onClose={toggleModal}
+        />
+      )}
```
Comment (axelhunn) [inline src/index.js:101]:
```suggestion
      <AcqKeyboardShortcutsModal
        open={isOpen}
        onClose={toggleModal}
      />
```
See my comment here https://github.com/folio-org/stripes-acq-components/pull/494#discussion_r632631446

File: src/index.js [hunk 2]
```diff
@@ -5,63 +5,108 @@ import {
...
-    stripes: PropTypes.object,
-    history: PropTypes.object,
-  }
-
-  focusSearchField = () => {
+const Finance = ({
+  match,
+  showSettings,
+  location,
+}) => {
```
Comment (axelhunn) [inline src/index.js:33]: Now that it's a functional component, could we use [router hooks](https://reactrouter.com/web/api/Hooks) to get `match` and  `location`?
Not to drill it down the props

File: src/index.js [hunk 3]
```diff
@@ -5,63 +5,108 @@ import {
...
+          </AppContextMenu>
           <Switch>
             <Route
-              path={`${this.props.match.path}`}
-              render={() => <Main {...this.props} />}
+              path={`${match.path}`}
+              render={() => <Main match={match} />}
             />
-            <Route component={() => { this.NoMatch(); }} />
           </Switch>
```
Comment (axelhunn) [inline src/index.js:94]: why Switch one Root?

**Review by axelhunn — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (AndreiShumski2) [reply src/index.js:98]: Actually, `open={isOpen}` will always display modal

Comment (axelhunn) [reply src/index.js:98]: Yes, you're right. `<KeyboardShortcutsModal>` has this
https://github.com/folio-org/stripes-components/blob/a804cdf3d521fe3989e8c4545ccb442811607631/lib/KeyboardShortcutsModal/KeyboardShortcutsModal.js#L64

Probably it's a mistake too...

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**